### ISSUE TITLE
Object3D vs. PhysicalObject properties?

### ISSUE THREAD

**Issue by taiya** [google-research/kubric#120]
Why are `scale` and `bounds` a PhysicalObject property?
I was expecting them to be both Object3Ds

**Comment by Qwlouse** [comment]
Because `Camera` and `Light` inherit from `Object3D` and they have a position and location but no scale and bounds.

**Comment by taiya** [comment]
True, but blender still has a scale/bounds property for this object.
This is mostly used in the viewer to determine the "visual" Gizmo?
(bounds could just be "0")
